apps/prairielearn/src/ee/pages/instructorAiGenerateJobReview/instructorAiGenerateJobReview.html.ts

```diff
@@ -0,0 +1,70 @@
+import { formatDate } from '@prairielearn/formatter';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { JobStatus } from '../../../components/JobStatus.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobReviewPage({
+  resLocals,
+  genJobs,
+}: {
+  resLocals: Record<string, any>;
+  genJobs: Job[];
+}) {
+  return html` <!doctype html>
```

Comment:
```suggestion
  return html`
    <!doctype html>
```

Reply:
changed in 3c8bafd1470728a8c79b488213de1dfae0aed79e